[nextra] Change TS SDK Quickstart content (#428)
aptos-labs/developer-docs: apps/nextra/pages/en/build/sdks/ts-sdk/quickstart.mdx

# TypeScript SDK Quickstart

<Callout type="info">

> The complete code example can be found at the bottom of this page!

Reviewer: ```suggestion
The complete example code can be found at [Full Quickstart Code](#Full_Quickstart_Code) at the bottom of the page.
```

> </Callout>

Reviewer: Put a link to the anchor

<Steps>

### Initialize A Project

> This will initialize a typescript package with `index.ts`

Reviewer: ```suggestion
This will initialize a typescript package with `quickstart.ts`
```

```bash npm2yarn

> npm init && npm add -D typescript @types/node ts-node && npx tsc --init && mkdir src && echo 'console.log("Hello, TypeScript!");' > src/index.ts

Reviewer: let's name all these `quickstart.ts` to match the headings

> npm init && npm add -D typescript @types/node ts-node && npx tsc --init && mkdir src && echo 'console.log("Hello, TypeScript!");' > src/index.ts

Reviewer: ```suggestion
npm init && npm add -D typescript @types/node ts-node && npx tsc --init && mkdir src && echo 'async function example() { console.log("Running example!")}; example()' > src/quickstart.ts
```

```

### Test Initialization

To test if you have initialized the package correctly run:

```bash npm2yarn

> npm ts-node src/index.ts

Reviewer: This command doesn't work for me

Reviewer: This is supposed to be `npx ts-node`

> npm ts-node src/index.ts

Reviewer: ```suggestion
npx ts-node src/quickstart.ts
```
